.github/workflows/audit_service_tags.yml

```diff
@@ -88,31 +89,17 @@ jobs:
             docker compose -f docker-compose.test.yml run -e CHANGED_FILES=${{ env.CHANGED_FILES }} web bash \
               -c "CI=true DISABLE_BOOTSNAP=true bundle exec rake service_tags:audit_controllers_ci"
 
-  assign-failure-label:
-    if: failure()
-    needs: [audit_service_tags]
-    runs-on: ubuntu-latest
-    permissions: write-all
-    steps:
-      - uses: actions/checkout@v4
-
       - name: Add Failure label
+        if: failure()
         uses: actions-ecosystem/action-add-labels@v1
         with:
           fail_on_error: true
           number: ${{ github.event.pull_request.number }}
           labels: |
             audit-service-failure
 
-  remove-failure-label:
```

Comment:
Removed this job and added the job as an additional step to prevent loading actions/checkout@v4 again